zeebe/qa/integration-tests/src/test/java/io/camunda/zeebe/it/authorization/rest/AuthorizationAddPermissionAuthorizationIT.java

```diff
@@ -5,7 +5,7 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.zeebe.it.authorization;
+package io.camunda.zeebe.it.authorization.rest;
```

Comment:
🔧 As I said in my other comment, I will remove most of these tests. Not sure which one will be merged first 😄